static/js/admin.js

```diff
@@ -759,13 +765,20 @@ function _setup_page() {
                 full_name: JSON.stringify(full_name.val()),
             };
 
+            if (owner.val() !== undefined && owner.val() !== "") {
+                url = "/json/bots/" + email;
```

Comment:
Probably no good as a lint rule since we can't be confident the variable will be called email.